src/app/docs/button/button-docs.component.html

```diff
@@ -0,0 +1,196 @@
+              Code
+            </p>
+            <p>
+              <sky-code>&lt;button type=&quot;button&quot; class=&quot;sky-btn sky-btn-link-inline&quot;&gt;Inline link&lt;/button&gt;</sky-code>
+            </p>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            <div style="margin-bottom: 10px;">
+              <button type="button" class="sky-btn sky-btn-primary sky-btn-disabled">Primary</button>
+            </div>
+            <div style="margin-bottom: 10px;">
+              <button type="button" class="sky-btn sky-btn-default sky-btn-disabled">Secondary</button>
+            </div>
+            <div style="margin-bottom: 10px;">
+              <button type="button" class="sky-btn sky-btn-link sky-btn-disabled">Link</button>
+            </div>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Usage
+            </p>
+            <p>
+              The disable class disables all button types. The most common use case is to disable actions until users complete prerequisite tasks.
+            </p>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Code
+            </p>
+            <p>
+              <sky-code>class="sky-btn-disabled"</sky-code>
+            </p>
+          </td>
+        </tr>
+      </table>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="UX guidelines"
+    >
+      <sky-docs-design-guideline
+        heading="Button text"
+      >
+        <p>
+          For buttons with only one possible direct object (such as <strong>Add</strong> buttons in record list toolbars), use the following format for button text:
+        </p>
+        <p>
+          <strong>&lt;Verb&gt;</strong>
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/button/buttons-usage-1.png"
+          >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline>
+        <p class="sky-docs-text-block">
+          For buttons with multiple possible direct objects (such as <strong>Add alert</strong> buttons in page summary toolbars), use the following format for button text:
+        </p>
+        <p>
+          <strong>&lt;Verb&gt; &lt;direct object&gt;</strong>
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/button/buttons-usage-2.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline>
+        <p class="sky-docs-text-block">
+          Additional button text guidelines:
+        </p>
+        <ul>
+          <li>
+            Use sentence-style capitalization.
+          </li>
+          <li>
+            Make the direct object singular or plural based on whether the action lets users act on one or multiple items.
```

Comment:
The wording here seems awkward to me. What do you think of changing this to something:

Use singular or plural direct objects to indicate whether the button acts on one or multiple items.

Do you think that's any clearer?

Reply:
I think it is and implemented the suggestion.